book/05-distributed-git/sections/contributing.asc

```diff
@@ -705,8 +705,33 @@ You can set each value separately with a series of `git config` commands, or you
   sslverify = false
 -----
 
-If your IMAP server doesn't use SSL, the last two lines probably aren't necessary, and the host value will be `imap://` instead of `imaps://`.
```

Comment:
Looks like your text editor made a smart quote here. Can you change all the `’` characters to `'`s?